apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -111,7 +111,7 @@ export class DelayedJobsController {
         job.isAcknowledged = attributes.isAcknowledged;
         await job.save();
 
-        const entityName = (job.metadata as { entity_name?: string }).entity_name;
+        const entityName = job.metadata ? (job.metadata as { entity_name?: string }).entity_name : null;
```

Comment:
These should not be relying on any particular structure to the metadata. I think this would be cleaner:
```
const entityName = job.metadata?.entity_name ?? "";
```

Reply:
I couldn't use it that way because I had this error: 
Property 'entity_name' does not exist on type 'object'.ts(2339)

Because the definition is like this:
```
 @AllowNull
  @Column(JSON)
  metadata: object | null;
```
Some options to fix it:
  
`  const entityName = (job.metadata as any)?.entity_name ?? "";`

or:
```
@AllowNull
@Column(JSON)
metadata: any | null;
```